src/analysis/sensitivity.jl

```diff
@@ -53,8 +69,29 @@ NamedDimsArray, of min, mean, median, max, std, and cv summary statistics.
    GSA-cvd
    Combining variance- and distribution-based global sensitivity analysis
    https://github.com/baronig/GSA-cvd
+
+3. Puy, A., Lo Piano, S., & Saltelli, A. 2020.
+   A sensitivity analysis of the PAWN sensitivity index.
+   Environmental Modelling & Software, 127, 104679.
+   https://doi.org/10.1016/j.envsoft.2020.104679
+
+4. https://github.com/SAFEtoolbox/Miscellaneous/blob/main/Review_of_Puy_2020.pdf
+
+# Extended help
+Pianosi and Wagener have made public their review responding to a critique of their method
+by Puy et al., (2020). A key criticism by Puy et al. was that the PAWN method is sensitive to its
+tuning parameters and thus may produce biased results. The tuning parameters referred to are
+the number of samples (\$N\$) and the number of conditioning points - \$n\$ in Puy et al., but
+denoted as \$S\$ here.
+
+Puy et al., found that the ratio of \$N\$ (number of samples) to \$S\$ has to be sufficiently high
+(\$N/S > 80\$) to avoid biased results. Pianosi and Wagener point  out this requirement is not
+particularly difficult to meet. Using the recommended value (\$S := 10\$), a sample of 1024 runs
+(small for purposes of Global Sensitivity Analysis) meets this requirement (\$1024/10 = 102.4\$).
+Additionally, lower values of \$N/S\$ is more an indication of faulty experimental design moreso
+than any deficiency of the PAWN method.
 """
-function pawn(X::T1, y::T2, factor_names::Vector{String}; S::Int64=10)::NamedDimsArray where {T1<:AbstractMatrix{<:Real},T2<:AbstractVector{<:Real}}
+function pawn(X::AbstractMatrix{<:Real}, y::AbstractVector{<:Real}, factor_names::Vector{String}; S::Int64=10)::NamedDimsArray
```

Comment:
Following the style guide, this line shouldn't be longer than 92 characters

Reply:
Yes okay - I had 100 as the max upper but happy to conform to 92.